tests/test_md.py

```diff
@@ -46,7 +46,7 @@ def test_fwd_mode(self):
             lambda_offset_idxs,
             p_scale=3.0,
             # cutoff=0.5,
-            cutoff=100.0
```

Comment:
Indicates we may need to modify any instances where cutoff may have been `None` or very large

Reply:
This was accidentally introduced originally, and I'm restoring it to something more reasonable so we can test different cutoff.